### PR TITLE
Update build-everything.yml

### PR DESCRIPTION
**Pull request by timcappalli**
Ignore non-spec files

### CODE REVIEW

File: .github/workflows/build-everything.yml [hunk 1]
```diff
@@ -3,8 +3,29 @@ name: publish
...
+      - README.md
+      - CHANGELOG.md
+      - CODE_OF_CONDUCT.md
+      - LICENSE
+      - SECURITY.md
+      - '.github/**'
+      - Makefile
+      - contributing.md
+      - .gitignore
+
```
Comment (FragLegs) [inline .github/workflows/build-everything.yml:16]: Add `working-group-charter.md`

File: .github/workflows/build-everything.yml [hunk 2]
```diff
@@ -3,8 +3,29 @@ name: publish
...
+    paths-ignore: 
+      - README.md
+      - CHANGELOG.md
+      - CODE_OF_CONDUCT.md
+      - LICENSE
+      - SECURITY.md
+      - '.github/**'
+      - Makefile
+      - contributing.md
+      - .gitignore
```
Comment (FragLegs) [inline .github/workflows/build-everything.yml:28]: Add `working-group-charter.md`

**Review by FragLegs — CHANGES_REQUESTED** [review]

File: .github/workflows/build-everything.yml (hunk 1 above)
Comment (timcappalli) [reply .github/workflows/build-everything.yml:17]: done

File: .github/workflows/build-everything.yml (hunk 2 above)
Comment (timcappalli) [reply .github/workflows/build-everything.yml:29]: done

**Review by FragLegs — APPROVED** [review]

**Pull request merged**